shared/src/common/enums/enums.ts

```diff
@@ -5,5 +5,6 @@ export { HttpCode, HttpHeader, HttpMethod } from './http/http';
 export {
   MasterValidationMessage,
   MasterSignUpValidationRule,
```

Comment:
MasterSignUpValidationRule rename to `MasterValidationRule` and keep all rules (sign-in and sign-up) in one place